docs/dataset.md

```diff
@@ -174,6 +192,14 @@ mmtracking
 │   │   │   ├── frames (the unzipped folders)
 │   │   │   │   ├── 0-6LB4FqxoE_0
 │   │   │   │   ├── 07Ysk1C0ZX0_0
+│   │
+│   ├── otb100
+│   │   ├── Basketball.zip
```

Comment:
Put all zip files in a folder `zip_data`
Please also modify the scripts and dcos